Islandora/islandora
# Multiple breadcrumb parent fields

**GitHub Issue**: N/A

# What does this Pull Request do?

- Exposes the breadcrumbs configuration in a form page.
- Allows the breadcrumbs builder to use multiple entity reference fields for finding parents
- Fixes a breadcrumbs builder error when the 'includeSelf' configuration is true.

# What's new?

* Changes islandora_breadcrumbs.breadcrumbs schema to allow multiple reference fields instead of just one. Includes an update hook.
* Changes the breadcrumbs builder to use multiple reference fields the update config schema allows
* Added a form for updating breadcrumbs settings.
* Fixed an error that was causing the site to crash when the includeSelf option was true.
* Does this change add any new dependencies? No.
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? _Run the Drupal database update to execute the update hook._
* Could this change impact execution of existing code? No.

# How should this be tested?

* Create some nodes with the member_of field and verify the breadcrumbs work as expected
* Apply the PR
* Run the Drupal update (e.g. `drush updatedb -y`)
* Ensure the breadcrumbs are still working
* Change some settings on the new form @ `/admin/config/islandora/breadcrumbs`
* Ensure the changes work for the breadcrumbs
* Add another entity reference field
* Add that field to the breadcrumbs configuration
* Populate that field on a node and verify it is used for building breadcrumbs

# Documentation Status

* Does this change existing behaviour that's currently documented? No.
* Does this change require new pages or sections of documentation? I don't think so.
* Who does this need to be documented for?

# Interested parties
@Islandora/committers, esp. @jordandukart
